### PR TITLE
[CI:DOCS] Add image not found info to troubleshooting

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Addresses: https://bugzilla.redhat.com/show_bug.cgi?id=2139191

We have a timing issue when doing a build or pull of an image and a second process is removing them at the same time.  This is not a window we're going to close now, so I'm adding a note to the troubleshooting guide to document it.

[NO NEW TESTS NEEDED]

#### Does this PR introduce a user-facing change?

```release-note
NOne
```

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
This should be called, don't shoot yourself in the foot rule.

File: troubleshooting.md [hunk 1]
```diff
@@ -1344,3 +1344,45 @@ $ podman run --rm \
...
+Storing signatures
+Error: error creating build container: error locating pulled image "registry.access.redhat.com/ubi8-minimal:latest" name in containers storage: registry.access.redhat.com/ubi8-minimal:latest: image not known
+```
+
+#### Solution
+The general cause for this is a timing issue.  To make Podman commands as
+efficient as possible, read and write locks are only established for critical
+sections within the code.  When pulling an image from a repository, a copy of
+that image is first written to local storage using a write lock.  This lock is
+released before the image is then acquired/read.  If a second user does a
```
Comment (vrothberg) [inline troubleshooting.md:1381]:
```suggestion
released before the image is then acquired/read.  If a another process executes a
```

To not give the impression of it being a multi-tenancy problem.

**Review by vrothberg — COMMENTED** [review]
LGTM besides the nit.

File: troubleshooting.md [hunk 2]
```diff
@@ -1344,3 +1344,45 @@ $ podman run --rm \
...
+harmful command such as `podman system prune –all` or `podman system reset`
+or `podman rmi –all`, between the time the image is written and before the
+first process can acquire it, this type of `image not known` error can arise.
```
Comment (Luap99) [inline troubleshooting.md:1384]:
```suggestion
harmful command such as `podman system prune --all` or `podman system reset`
or `podman rmi --all`, between the time the image is written and before the
first process can acquire it, this type of `image not known` error can arise.
```
small nit but these things cause problems when copy pasting

Comment (TomSweeneyRedHat) [reply troubleshooting.md:1384]: Thanks for the eagle eye @Luap99 , that's exactly what happened, I wrote this up in google docs first and the cut/paste wiped out the extra dash.  @vrothberg thanks for yours too, I'll touch up both shortly.

**Comment by TomSweeneyRedHat** [comment]
Repushed

**Pull request merged**